proto/dream11/od/dto/v1/service.proto

```diff
@@ -22,8 +22,8 @@ message ServiceDefinition {
 }
 
 message ServiceIdentifier {
-  string name = 1;
-  string version = 2;
+  string service_name = 1;
```

Comment:
why this change? older names look good to me

Reply:
we have used this new name in deployer proto files. is easier to change name here than in deployer as it might be used in multiple places.